src/hooks/useFirestoreConnect.ts

```diff
@@ -40,8 +60,11 @@ export interface SparkleRFQConfig extends ReduxFirestoreQuerySetting {
  * @see ValidFirestoreKeys
  * @see ReduxFirestoreQuerySetting
  */
-export const useSparkleFirestoreConnect = (config: SparkleRFQConfig[]) =>
-  useFirestoreConnect(config);
+export const useFirestoreConnect = (
+  config:
+    | (SparkleRFQSetting | SparkleDocRFQSetting)
+    | (SparkleRFQSetting | SparkleDocRFQSetting)[]
+) => _useFirestoreConnect(config);
```

Comment:
Why the need for the explicit `|` here? What's the intent?

(and if we are going to keep it, we should make a type that wraps this up so we don't need to duplicate this pattern across the codebase)